p2p/switch.go

```diff
@@ -416,6 +467,75 @@ func (sw *Switch) stopAndRemovePeer(peer Peer, reason interface{}) {
 		// Removal of the peer has failed. The function above sets a flag within the peer to mark this.
 		// We keep this message here as information to the developer.
 		sw.Logger.Debug("error on peer removal", ",", "peer", peer.ID())
+
+		// check if we removed sentinel, if so, alert metrics
+		splitStr := strings.Split(sw.SentinelPeerString, "@")
+		if len(splitStr) > 1 {
+			sentinelIDConv := ID(splitStr[0])
+			if err := validateID(sentinelIDConv); err == nil {
+				if peer.ID() == sentinelIDConv {
+					sw.mevMetrics.SentinelConnected.Set(0)
```

Comment:
if the sentinel peer was removed, the metrics should be informed. 
It seems that mevMetrics.SentinelConnected will be set to 0 in case of removal NOT succeeded. If I understand the SentinelConnected flags correctly, this should be done in if `(sw.peers.Remove(peer)) ` branch.

Also, currently `sentinelConnected`  (matrics.Gauge)  is not used anywhere, so I'm not sure how holding the wrong info could impact  the metrics - but it seams peripheral.